config/initializers/devise.rb

```diff
@@ -27,7 +27,7 @@
   config.mailer_sender = "please-change-me-at-config-initializers-devise@example.com"
```

Comment:
Is the notify mailer sender address configured in our Notify tenant? Now we're using Notify, do we have to change this or is it effectively unused/removable/renamable as `"i-am-irrelevant-please-ignore-me@example.com"`?

Reply:
Ah yeah, found the reply-to in Notify's settings. Maybe we should remove this.